modules/sdp/LocalSdpMunger.js

```diff
@@ -175,11 +177,14 @@ export default class LocalSdpMunger {
                     const streamAndTrackIDs = ssrcLine.value.split(' ');
 
                     if (streamAndTrackIDs.length === 2) {
-                        const streamId = streamAndTrackIDs[0];
+                        let streamId = streamAndTrackIDs[0];
                         const trackId = streamAndTrackIDs[1];
 
-                        ssrcLine.value
-                            = `${streamId}-${pcId} ${trackId}-${pcId}`;
+                        // eslint-disable-next-line max-depth
+                        if (streamId === '-') {
```

Comment:
can you put the old comment here which describes the firefox case?

Reply:
also the old code which was creating id which was media specific was better than just the endpoint id

Reply:
I don't know about the lipsync logic, but it's been broken anyway maybe there's a better way of doing it